misc/alert_rules.json

```diff
@@ -116,6 +116,72 @@
     "rule": "%services.service_status = \"2\"",
     "name": "Service critical"
   },
+  
+   {
+    "rule": "%syslog.timestamp >= %macros.past_5m && %syslog.priority ~ \"alert\"",
+    "name": "Syslog, received Alert Priority Message"
+  },
+  
+    {
+    "rule": "%syslog.timestamp >= %macros.past_5m && %syslog.priority ~ \"emerg\" || %syslog.priority ~ \"emergency\"",
+    "name": "Syslog, received Emergency Priority Message"
+  },
+  
+     {
+    "rule": "%syslog.timestamp = %macros.past_5m && %syslog.msg ~ \"@arp table is full@\"",
+    "name": "Syslog, ARP table is full check on device "
+  },
+  
+   {
+    "rule": "	%sensors.sensor_type ~ \"upsAdvBatteryReplaceIndicator\" && %sensors.sensor_current = \"2\"",
+    "name": "APC UPS Battery Needs Replacement"
+  },
+  
+  {
+    "rule": " %sensors.sensor_current = \"3\" && %sensors.sensor_type = \"upsBasicOutputStatus\"",
+    "name": "APC UPS Switched to Battery Power"
+  },
+  
+  {
+    "rule": " %sensors.sensor_current = \"10\" && %sensors.sensor_type = \"upsBasicOutputStatus\"",
+    "name": "APC UPS in Hardware Failure Bypass Mode"
+  },
+  
+  {
+    "rule": " %sensors.sensor_current = \"16\" && %sensors.sensor_type = \"upsBasicOutputStatus\"",
+    "name": "APC UPS in Emergency Static Bypass Mode"
+  },
+  
+  {
+    "rule": " %sensors.sensor_current = \"12\" && %sensors.sensor_type = \"upsBasicOutputStatus\"",
+    "name": "APC UPS in Smart Trim Mode"
+  },
+  
+   {
+    "rule": " %sensors.sensor_oid = \".1.3.6.1.4.1.11.2.14.11.1.2.6.1.4.2\" && %sensors.sensor_current = \"2\"",
+    "name": "HP Procurve Bad PSU #1"
+  },
+  
+    {
+    "rule": " %sensors.sensor_oid = \".1.3.6.1.4.1.11.2.14.11.1.2.6.1.4.1\" && %sensors.sensor_current = \"2\"",
+    "name": "HP Procurve Fan Fault"
+  },
+  
+     {
+    "rule": " %sensors.sensor_oid = \".1.3.6.1.4.1.11.2.14.11.1.2.6.1.4.3\" && %sensors.sensor_current = \"2\"",
+    "name": "HP Procurve Bad PSU #2"
+  },
+
+   {
+    "rule": " %sensors.sensor_oid = \".1.3.6.1.4.1.11.2.14.11.1.2.6.1.4.4\" && %sensors.sensor_current = \"2\"",
+    "name": "HP Procurve Bad PSU #3"
+  },
+  
+     {
+    "rule": " %sensors.sensor_oid = \".1.3.6.1.4.1.11.2.14.11.1.2.6.1.4.5\" && %sensors.sensor_current = \"2\"",
```

Comment:
The PSUs might be better to do `~ \".1.3.6.1.4.1.11.2.14.11.1.2.6.1.4.[2-5]\"`

Reply:
so remove each one and make just one rule for the PSU alert rule?

Reply:
Yup